### ISSUE TITLE
Start HTTPS only

### ISSUE THREAD

**Issue by gouyez** [inconshreveable/ngrok#656]
Hi,
When I start an HTTP tunnel, ngrok automatically starts forwarding on both HTTP and HTTPS.
Is there a way to only start one of them?

**Comment by JennaSys** [comment]
Per the docs, set option bind-tls=true for just https and bind-tls=false for just http:
`ngrok http -bind-tls=true site.dev:80`

**Comment by mrchomik892** [comment]
I don't know, but you can create redirect from http to https?
It's stupid question.

**Comment by gouyez** [comment]
> I don't know, but you can create redirect from http to https?
> It's stupid question.

Yes, I could do that, but it wouldn't help, ngrok gives 4 tunnels (on the free plan), HTTP takes two of them as it uses one for HTTP and the other for HTTPS, which leaves me with only two free tunnels.

I did redirect traffic to different ports using ProxyPass (apache), now everything goes through port 80, and then get redirected to the desired port through apache

**Comment by russorat** [comment]
Thank you for opening this issue! As of April 2016, the ngrok service has permanently moved to https://ngrok.com/. To join the community and file bug reports or feature enhancements for the ngrok agent, please go to https://github.com/ngrok/ngrok.

This repository is no longer actively maintained and will be archived soon to reduce confusion with the latest ngrok versions. We thank you for the continued support of ngrok and look forward to seeing you over at [ngrok.com](https://ngrok.com/) and the [ngrok Community Repo](https://github.com/ngrok/ngrok).
